### ISSUE TITLE
wave file

### ISSUE THREAD

**Issue by ghost** [KaiDrange/OctaChainer#10]
I cannot create a wave file  I have tried everything,,  when I hit create it create a tiny wave  file that does not work, is this a bug?  I am on OS X  I try in serria and Catalina, thank you

**Comment by jamesdmorgan** [comment]
I'm seeing the same thing the .wav file is 44 bytes. I tried with quite big / long samples that I wanted to combine and add slices. I noticed that I couldn't preview the long samples but if I added a clap or short sample I could preview it and the wav file increases. Feels like its choking on the larger ones and unable to add them. I'm on OSX Mojave btw

**Comment by jamesdmorgan** [comment]
I think its https://github.com/KaiDrange/OctaChainer/issues/8#issue-398463435 will have a play around and report back

**Comment by jamesdmorgan** [comment]
Yep it was spaces in my files - ableton 10 adds datetime stamp at the end of consolidated wavs with spaces...

**Comment by KaiDrange** [comment]
I was not able to reproduce the issue, so either this was already fixed when I tested, or perhaps the datetime stamp had some other character or white space that caused a problem? I changed how file names are handled in v1.3.1. If the root cause for the issue was the file names, it is most likely not an issue any more. I'll keep this issue open so that I remember to take a second look, just in case.

There is currently no error handling on file IO at all. That needs to be improved so that reporting and fixing these kinds of errors will be easier.
